### PR TITLE
add plot scripts

### PR DESCRIPTION
**Pull request by lightingghost**
Hi google research owners,

I am submitting a pull request adding the plot scripts. Please review.

### CODE REVIEW

**Comment by skearnes** [comment]
Sorry, I just realized this PR existed. I don't get an email when a new one is created...

File: mol_dqn/chemgraph/dqn/deep_q_networks.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def _build_single_q_network(self, observations, head, state_t, state_tp1,
     # the asymptotic relative efficiency of the l1 loss estimator is better
     # for heavy-tailed distributions.
     errors = tf.where(
-        tf.abs(td_error) < 1.0, tf.square(td_error),
+        tf.abs(td_error) < 1.0, tf.square(td_error) * 0.5,
```
Comment (skearnes) [inline mol_dqn/chemgraph/dqn/deep_q_networks.py]: tf.losses.huber_loss?

Comment (skearnes) [reply mol_dqn/chemgraph/dqn/deep_q_networks.py]: Note that your code doesn't _exactly_ match the huber loss right now; you have a strict inequality instead of <=

Comment (skearnes) [reply mol_dqn/chemgraph/dqn/deep_q_networks.py]: Resolved; this is equivalent.

**Comment by skearnes** [comment]
Thanks Zhenpeng! I'll work on getting this merged.

**Comment by lightingghost** [comment]
Thanks Steve!
